### ISSUE TITLE
Electricitymap graph on left side flickering ON/OFF

### ISSUE THREAD

**Issue by StephanVerbeeck** [electricitymap/electricitymap-contrib#4106]
![graphs](https://user-images.githubusercontent.com/40359489/167360478-b0a83ac3-f51c-435a-916f-cb50fe434898.png)

# Observed:
When hovering the mouse pointer above the graph that shows the different types of electricity production over the last 24 hours the message "Some of the data for this region is estimated. Read more about our estimation methodology here." is shown.
This is one of multiple CONDITIONAL messages that are ONLY shown if a specific type of energy is pointed at.
However the display of this message SHIFTS all the graphs downward with the visual HEIGHT of that message making that the mouse cursor is no longer above the same energy type making the message disappear again.
This causes the entire display to FLICKER like hell rendering the entire website useless.
Whoever made the change of adding these message did not do any testing! (it should never have happened like that)

# Expected:
Display of conditional messages may NOT change the position of the graph to which these messages belong.

# How to fix:
There are 4 possible fixes to this horrible visual bug.
1) remove all these messages
2) make a DIV with fixed dimensions in which these messages appear so that the space is always reserved (even when the message is not shown)
3) make the messages popup in a separate window that is not above the graph.
4) move all the conditional message BELOW all the graphs instead of ABOVE the graphs (like it is now).

# Note:
fix 4 is the easiest to implement (15 minutes work including testing).
Please test changes,  This is an error that should never have been committed.

**Comment by VIKTORVAV99** [comment]
This is a duplicate of #3807 and a fix has already been discussed there.

**Comment by StephanVerbeeck** [comment]
> This is a duplicate of #3807 and a fix has already been discussed there.

Correct, though my proposed solution #4 is much simpler

**Comment by madsnedergaard** [comment]
Thanks for the detailed report and solutions shared here!
We already have an open issue for this in #3807 - but would be great if you or anyone else wants to fix it?

I'll close this issue to keep information in one place, but we appreciate the additional info you gave here :)

PS. regarding the "Please test changes, This is an error that should never have been committed." comment:

I agree that this is an annoying error, and the fact that it's still there a long time after we discovered it is also a bit annoying. 
We are however only a small team working on this product and we're therefore quite dependent on the open-source community and the amazing amount of work they do - and as such prioritisation isn't always about what's most urgent, but about what people are happy to contribute with.

Long story short, mistakes happen and not everything is caught in testing - and we'd rather release often and get as many contributions in, than spending time in QA :)
